strawberry/cli/commands/export_schema.py

```diff
@@ -18,7 +20,19 @@ def export_schema(
             "Works the same as `--app-dir` in uvicorn."
         ),
     ),
+    output: Path = typer.Option(
+        None,
+        "--output",
+        help="File to save the exported schema. If not provided, prints to console.",
+    ),
 ) -> None:
     schema_symbol = load_schema(schema, app_dir)
 
-    print(print_schema(schema_symbol))  # noqa: T201
+    schema_text = print_schema(schema_symbol)
+
+    if output:
+        with Path.open(output, "w") as file:
+            file.write(schema_text)
```

Comment:
```suggestion
        Path(output).write_text(schema_text)
```

I think this should be enough 😊